src/app/components/PublisherCard/index.tsx

```diff
@@ -32,14 +32,14 @@ export default function PublisherCard({
       className={classNames(
         isSmall ? "p-2" : "flex-col justify-center p-4",
         isCard && "drop-shadow rounded-lg mt-4",
-        !image && "h-24",
+        !image && "h-16",
         "flex items-center bg-white dark:bg-surface-02dp"
       )}
     >
       {image && (
         <img
-          className={`m-2 shrink-0 object-cover rounded-lg ${
-            isSmall ? "w-14 h-14 mr-4" : "w-20 h-20"
+          className={`ml-1 mr-2 shrink-0 object-cover rounded-lg ${
```

Comment:
Can we change to rounded-md (it's more harmonous with the box rounding when image is smaller)?

Reply:
I updated. Like this now?